File: frontend/src/types/patient.ts
export interface Patient {
  id: number;
  first_name: string;
  last_name: string;
  birth_date?: string | null;
  phone?: string | null;
  email?: string | null;
  address?: string | null;
  doctor_id?: number | null;
}

export interface CreatePatientDTO {
  first_name: string;
  last_name: string;
  birth_date?: string;
  phone?: string;
  email?: string;
  address?: string;
  doctor_id?: number | null;
}

File: backend/app/models/patient.py
from sqlalchemy import Column, Integer, String, Date
from app.database.connection import Base


class Patient(Base):
    __tablename__ = "patients"

    id = Column(Integer, primary_key=True, index=True)
    first_name = Column(String, nullable=False)
    last_name = Column(String, nullable=False)
    birth_date = Column(Date)
    phone = Column(String)
    email = Column(String, unique=True)
    address = Column(String)
